backend/src/test/java/com/benine/backend/camera/ipcameracontrol/IpcameraTest.java

```diff
@@ -84,6 +84,25 @@ public final void testGetPosition() throws CameraConnectionException {
     assertEquals(180, res.getTilt(), 0.000001);
   }
   
+  @Test
+  public final void testGetPositionTwice() throws CameraConnectionException {
+    setCameraBehaviour("APC", "aPC80008000");
+    Position res = camera.getPosition();
+    res = camera.getPosition();
+    assertEquals(0, res.getPan(), 0.000001);
+    assertEquals(180, res.getTilt(), 0.000001);
+  }
+  
+  @Test
+  public final void testGetPositionTwice3Seconds() throws CameraConnectionException, InterruptedException {
+    setCameraBehaviour("APC", "aPC80008000");
+    Position res = camera.getPosition();
+    Thread.sleep(2100); 
```

Comment:
This slows down testing by a lot if this happens to often.  Maybe you can make the amount of time before it times out an attribute (maybe even get it from the config) so you can set it to a really small amount when testing.